platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -550,9 +550,9 @@ with the `MGLSymbolStyleLayer.text` property.
 
  Key | Value Type
  --- | ---
- `MGLFontNamesAttribute` | `NSExpression`
- `MGLFontScaleAttribute` | `NSExpression`
- `MGLFontColorAttribute` | `NSExpression`
+ `MGLFontNamesAttribute` | An `NSExpression` evaluating to an `NSString` array.
+ `MGLFontScaleAttribute` | An `NSExpression` evaluating to a constant `NSNumber` value.
```

Comment:
Does the expression have to be constant? If not, remove “constant”, because it’s a bit misleading.